fixieai/agents-ts/src/__tests__/serve.ts

```diff
@@ -68,19 +68,57 @@ describe('server starts', () => {
 
     expect(response.statusCode).toBe(404);
     expect(response.body).toBe(
-      'Function not found: function-does-not-exist. Functions available: roll, willThrowError',
+      'Function not found: function-does-not-exist. Functions available: roll, willThrowError, willThrowErrorAsync, rollAsync',
     );
   });
 
-  it('Function being called throws an error', async () => {
```

Comment:
This diff looks bigger than it is – I'm just adding two new tests. Two previously-existing tests are moved into this `describe` block.